### DISCUSSION TITLE
Is there a way to get rid of the black bars?

### DISCUSSION THREAD

**Discussion by eduardomart** [trvswgnr/bs5-lightbox#38] [Q&A] [answered]
Thank you

**Comment by christopherhealey** [comment, marked as answer]
You can target (.lightbox-carousel .carousel-item .ratio) in css and set background-color: transparent!important; that will work, only solution I've seen so far other than hacking the code.

**Comment by mynona** [comment]
Thx,that worked. Would be nice if the background and the bars would be configurable per se.
